modules/thrift/src/thrift/server/TFDServer.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 Meta
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include <thrift/transport/TServerTransport.h>
+
+namespace apache
+{
+namespace thrift
+{
+namespace transport
+{
+
+class TFDServer : public TServerTransport
+{
+
+      public:
```

Comment:
Can we please keep the indentation style consistent? (if not using two spaces?)

Mixing TABs and SPACEs is just confusing.

Reply:
This was formatted with the `.clang-format` in the Zephyr Project. Is it perhaps missing some rules?

Reply:
We will likely need a different clang-format file for the C++ code (AFAIK, clang-format treats C and C++ as the same language and does not allow you to specify different rules for them in the same file).

Besides that, the indentation style seems to be mixed and inconsistent -- there are multiple occasions where SPACEs and TABs are used together in the same line and also completely different indentation style (e.g. in `tests/lib/thrift/ThriftTest/src/server.hpp` using two spaces).

Reply:
Oh yea - I see what you mean now - the current `.clang-format` prepends a tab + 2 spaces to `public:`, `private:`, and `protected:`. That is nasty.